ticketauth/src/main/kotlin/dk/ufst/ticketauth/TicketAuth.kt

```diff
@@ -15,6 +15,7 @@ object TicketAuth {
 
     fun setup(config: TicketAuthConfig) {
         debug = config.debug
+        engine?.destroy()
```

Comment:
When the calling `Activity` goes to background there will technically be a leak in ticketauth until the `Activity` comes to foreground and calls the `setup` function again. I have to test if this is true. 

If we expose a destroy function to the client call, it would remove the cleaning up responsibility from the library and on to the client.

Reply:
I tested it out with LeakCanary and couldn't find a leak with the new changes in this PR.